viz_config.yml

```diff
@@ -7,7 +7,7 @@ date_display_tz: 'UTC'
 
 # view, dimensions
 bbox: [-83, 25, -67, 39]
-projection: "+proj=lcc +lat_1=36.16666666666666 +lat_2=34.33333333333334 +lat_0=33.75 +lon_0=-79 +x_0=609601.2192024384 +y_0=0 +ellps=GRS80 +datum=NAD83 +to_meter=0.3048006096012192 +no_defs"
+projection: "+init=epsg:2264"
```

Comment:
can you explain why we need to switch the projection?

Reply:
Didn't change, just using the epsg code rather than the fill proj4 string. I was worried that something about the proj parameters was causing the problem so I used the code to hit the database.